docs/introduction/composition.md

```diff
@@ -307,6 +307,16 @@ spec:
   compositeTypeRef:
     apiVersion: example.org/v1alpha1
     kind: CompositeMySQLInstance
+
```

Comment:
Could you add a one-liner update description at the top of this document similar to https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#revisions